config/envs/default.py

```diff
@@ -99,6 +99,9 @@ class DefaultConfig(object):
     FSD_SESSION_TIMEOUT_SECONDS = CommonConfig.FSD_SESSION_TIMEOUT_SECONDS
     WTF_CSRF_TIME_LIMIT = CommonConfig.WTF_CSRF_TIME_LIMIT
     CREATE_APPLICATION_ON_ACCOUNT_CREATION = False
+    ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK = getenv(
+        "ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK", True
```

Comment:
need to use `strtobool` here as string from env will always be considered truthy.

Reply:
thank you! have added this now 👍🏾